api.js

```diff
@@ -116,6 +116,19 @@ Api.prototype._run = function (files, _options) {
 	self.precompiler = new CachingPrecompiler(cacheDir, self.options.babelConfig);
 	self.fileCount = files.length;
 
+	var overwatch;
+	if (this.options.concurrency > 0) {
```

Comment:
Normally I would agree but with the current behavior, no matter what if the `--concurrency` flag is used the code behind `_runLimitedPool` is used.